### ISSUE TITLE
Release Error

### ISSUE THREAD

**Issue by SuperHero2010** [wmw-modding/wheres-my-editor#8]
I build "WME" done but I can't enter the app.

**Comment by ego-lay-atman-bay** [comment]
What do you mean by you can't enter the app?

**Comment by SuperHero2010** [comment]
When build is finished and enter the app, it shows an error and then disappears.

**Comment by ego-lay-atman-bay** [comment]
Well, what's the error?

**Comment by SuperHero2010** [comment]
Wait a minute.

**Comment by SuperHero2010** [comment]
Um... Just finished building it and before I could take a screenshot, it disappeared.
I just took this photo:
![10](https://github.com/wmw-modding/wheres-my-editor/assets/156646084/6e1b1db7-719d-4077-a98b-a8c1175fea39)

**Comment by ego-lay-atman-bay** [comment]
Judging by the fact that the icon doesn't show up, you probably didn't build it right.

**Comment by SuperHero2010** [comment]
Am I missing anything?

**Comment by ego-lay-atman-bay** [comment]
I have no idea. The best to do would probably just to run `main.py` instead of trying to buiod the exe.

**Comment by SuperHero2010** [comment]
buiod ---> build
Edit your comments

**Comment by SuperHero2010** [comment]
![11](https://github.com/wmw-modding/wheres-my-editor/assets/156646084/ef599c38-8497-4420-a4cd-7413c5b81434)
Here is it.

**Comment by ego-lay-atman-bay** [comment]
Oh yeah... that's just windows defender thinking it's a virus. Believe me, it's not. If you want to make sure, put it into https://virustotal.com, and it will tell you which antiviruses flag it as a virus, and if most say it's fine, then it's most likely fine.

As for getting rid of that popup, add it as an exception by following this: https://support.microsoft.com/en-us/windows/add-an-exclusion-to-windows-security-811816c0-4dfd-af4a-47e4-c301afe13b26 I recommend adding the whole WME folder.

Of course if you don't trust the exe, you can still just run the `src/main.py` file with python just fine.

**Comment by SuperHero2010** [comment]
Do you know my YouTube channel and watch my videos? I believe you. You don't have to tell me to believe you. But I don't know how to add the entire WME folder.

**Comment by SuperHero2010** [comment]
[@wheresmywatergameplay2004](https://youtube.com/@wheresmywatergameplay2004) use "MT Manager" to edit WMW levels and objects without use "Where's my Editor?".

**Comment by SuperHero2010** [comment]
It seems like you're too busy so I don't want to ask you and message you all the time.

**Comment by ego-lay-atman-bay** [comment]
> Do you know my YouTube channel and watch my videos? I believe you. You don't have to tell me to believe you. But I don't know how to add the entire WME folder.

Yes, I know about your youtube channel (I got norifications for you mentioning me in the descriptions).

> [@wheresmywatergameplay2004](https://youtube.com/@wheresmywatergameplay2004) use "MT Manager" to edit WMW levels and objects without use "Where's my Editor?".

I know, I was asking that specifically so I could tell them about where's my editor. It's a setup.

**Comment by SuperHero2010** [comment]
So have you watched my video yet?

**Comment by SuperHero2010** [comment]
So have you watched my video yet?

**Comment by SuperHero2010** [comment]
I build my edited "Where's my Editor" without use command: `python build.py` and with yml files in GitHub.
I know you already know this.

**Comment by SuperHero2010** [comment]
But why when I build WME done, it's shows `wmwpy-0.6.0-beta` without show `wmwpy-0.5.2-beta` (Even though I used the old file)?

**Comment by SuperHero2010** [comment]
If you want to download my edited WME is blocked by Chrome because of a virus, turn off your firewall and antivirus program.
https://github.com/SuperHero2010/wheres-my-editor/releases

**Comment by SuperHero2010** [comment]
Don't have time to post WME 2.3.4 and WME 2.4.0?

**Comment by SuperHero2010** [comment]
I need your help: When I create a `release` and `pyinstaller-build.yml` works, all WME MacOS, Ubuntu, Windows of `pyinstaller-build.yml` fail.

**Comment by SuperHero2010** [comment]
![Screenshot_20240715-123153](https://github.com/user-attachments/assets/057eb63b-9f18-46d7-9646-bc2a0e3179a4)

**Comment by ego-lay-atman-bay** [comment]
I just updated the release workflow to just be much better (and I now don't have to use a personal github token, which was the problem, but you don't need to do anything after merging the changes).

**Comment by SuperHero2010** [comment]
So what do I have to do to make the workflow: `pyinstaller-build.yml` work successfully?

**Comment by ego-lay-atman-bay** [comment]
I renamed that file to `release-assets.yml`, and once you use my new version, it should just work.

**Comment by SuperHero2010** [comment]
Before I try, I tell you this: Pay attention to the messages above, don't just pay attention to the messages below. You should reply to the messages above so I can know your answers to the messages above.

**Comment by ego-lay-atman-bay** [comment]
Huh?

**Comment by SuperHero2010** [comment]
It's okay. I'm just reminding you, because you didn't reply to my messages above.

**Comment by SuperHero2010** [comment]
Thank you, the problem is that it doesn't write the version of all WME.zip but that's okay.
Try my edited WME here (If you want to download older versions, turn off your computer's firewall and antivirus program): https://github.com/SuperHero2010/wheres-my-editor/releases

**Comment by SuperHero2010** [comment]
If you turn off firewall and antivirus program but Chrome still bans files, I'll show you how to turn it off.

**Comment by SuperHero2010** [comment]
Initially, I was planning to fix many of the remaining errors myself, but you fix them, that's it.

**Comment by SuperHero2010** [comment]
Can you replace `-` to `folder` into the `FillTexture` of all ducks?

**Comment by SuperHero2010** [comment]
These are the questions you didn't answer:
- So have you watched my video yet?
- But why when I build WME done, it's shows `wmwpy-0.6.0-beta` without show `wmwpy-0.5.2-beta` (Even though I used the old file)?
- Can you replace `-` to `folder` into the `FillTexture` of all ducks?

**Comment by ego-lay-atman-bay** [comment]
> * So have you watched my video yet?

I have watched some of your videos, I just have nothing to say about them.



> * But why when I build WME done, it's shows `wmwpy-0.6.0-beta` without show `wmwpy-0.5.2-beta` (Even though I used the old file)?

That's because I made wme use the latest released version when you create a release, and then use the latest dev version when you create a commit. The reason the version number is different from before, is because it used the latest wmwpy version (which is a seperate project, that's why it has it's own version number).

> * Can you replace `-` to `folder` into the `FillTexture` of all ducks?

Honestly I don't really want to do it, because it's not really needed.
